wgpu-core/Cargo.toml

```diff
@@ -46,3 +46,6 @@ gfx-backend-vulkan = { version = "0.4", features = ["x11"] }
 gfx-backend-dx12 = { version = "0.4.1" }
 gfx-backend-dx11 = { version = "0.4" }
 gfx-backend-vulkan = { version = "0.4" }
+
+[target.'cfg(any(target_os = "linux", target_os = "macos", target_os = "windows", target_os = "dragonfly", target_os = "freebsd"))'.dependencies]
```

Comment:
does battery simply not build on other platforms?

Reply:
It does not:

https://github.com/svartalf/rust-battery/blob/bed111f51f02d172f26ed00fa6a73ea51e084a60/battery/src/platform/mod.rs#L27-L28

Reply:
They could have just mocked it on their side, returning an Error...